neofetch

```diff
@@ -16,9 +16,8 @@ XDG_CONFIG_HOME="${XDG_CONFIG_HOME:-${HOME}/.config}"
 export LC_ALL=C
 export LANG=C
 
-# Set PATH to binary directories only
-# This solves issues with neofetch opening the pacman game.
-export PATH="/usr/sbin:/usr/bin:/sbin:/bin"
+# Add /usr/sbin and /sbin to PATH.
+export PATH="${PATH}:/usr/sbin:/sbin"
```

Comment:
I think this is better the other way around.
`PATH="/usr/sbin:/sbin:${PATH}"` since Neofetch will mostly interact with `/usr/sbin`.